cmd/coder/urls.go

```diff
@@ -14,13 +14,120 @@ import (
 )
 
 type urlsCmd struct{}
+type createURLCmd struct{}
+type delURLCmd struct{}
 
+// DevURL is the parsed json response record for a devURL from cemanager
 type DevURL struct {
+	ID     string `json:"id"`
 	URL    string `json:"url"`
 	Port   string `json:"port"`
 	Access string `json:"access"`
 }
 
+var urlAccessLevel = map[string]string{
+	"PRIVATE": "Only you can access",
+	"ORG":     "All members of your organization can access",
+	"AUTHED":  "Authenticated users can access",
+	"PUBLIC":  "Anyone on the internet can access this link",
```

Comment:
The values don't technically matter currently. Validation of access levels just checks user actions against the keys.